assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -486,6 +522,11 @@ describe("DiversionPage", () => {
       routeSegments: routeSegmentsFactory.build(),
     })
 
+    const intersection = "Avenue 1 & Street 2"
+    jest.mocked(useNearestIntersection).mockReturnValue({
+      ok: intersection,
+    })
```

Comment:
nit(non-blocking): would prefer the `ok` constructor from `fetchResult` instead of the object
```suggestion
    jest.mocked(useNearestIntersection).mockReturnValue(ok(intersection))
```